test/unit/mocks/FakeProjectMock.ts

```diff
@@ -55,6 +55,7 @@ export class FakeProjectMock {
 
   getExpectedProperties() {
     return {
+      'sonar.exclusions': DEFAULT_SONAR_EXCLUSIONS,
```

Comment:
We are now always setting `sonar.exclusions` (before, we weren't setting it when there was a `sonar-project.properties` file preset